### DISCUSSION TITLE
Whether all listed damage types in WeaponAsset.md can be used by magazine.

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#3084] [Q&A] [answered]
(https://github.com/SmartlyDressedGames/U3-Docs/blob/master/ItemAsset/MagazineAsset.md) According to this, there wasn't anything about whether magazines could also do Player_Damage_Bleeding, Stun_Zombie_Never and BladeIDs for example.

Could those lines, when listed under magazine's.dat files instead of the gun's .dat file, be read by the game? (Thus allowing for multipurpose guns/multipurpose mags/more customization)

**Comment by SDGNelson** [comment, marked as answer]
Sorry, not currently because MagazineAsset is not considered a "weapon"
